### PR TITLE
Cumulative Gridsome revamp

### PR DESCRIPTION
**Pull request by aaronsteers**
Do not merge. This PR is created for review and discussion purposes.

This PR represents the replacement of the legacy `main` branch with the new `gridsome-main` branch which is a trunk on its own.

**[Latest (successful) preview build here.](https://meltano-hub-gridsome.netlify.app/)**

Todo before merging:

- [Gridsome MVP Milestone](https://github.com/meltano/hub/milestone/3)

### CODE REVIEW

**Comment by aaronsteers** [comment]
@rwfeather - This test is failing but it's great to see actually. A nice referential integrity test. 🙏 

```
Extra Maintainers: {'cmbramwell', 'rangle', 'phillymedia', 'ragbadaskar', 'meowwolf', 'reachlocal'}
```

@pnadolny13 - Is this to be expected or do we think these maintainers should still have one or more registered plugins?

File: README.md
Comment (aaronsteers) [inline README.md:19]: Should we link to `CONTRIBUTING.md` here and merge this with the content there?

**Comment by pnadolny13** [comment]
> Hypothesis: We removed from `main` and that correctly got merged into `gridsome-main`, but `gridsome-main` requires more stringent culling of the `maintainers` list, which is why this popped up here and not in `main`.
> 
> Yes/no/maybe? 😄

@aaronsteers yes, we removed those in https://github.com/meltano/hub/pull/728 because the repos are no longer available.

**Comment by aaronsteers** [comment]
> Hypothesis: We removed from `main` and that correctly got merged into `gridsome-main`, but `gridsome-main` requires more stringent culling of the `maintainers` list, which is why this popped up here and not in `main`.

- Resolved in #750

File: src/layouts/Default.vue
Comment (aaronsteers) [inline src/layouts/Default.vue:21]: @alexmarple - This came in via a merge conflict from main. It was in `Head.html`, which we don't have in this branch. Feel free to relocate or delete as appropriate.

**Review by aaronsteers — COMMENTED** [review]
@cjohnhanson and @rwfeather -

@tayloramurphy will be the final approver here, but for my part, I'm _low-tech_ approving to ship the new site Monday, pending us successfully landing #840.

_Have a great weekend everyone and thanks for all the hard work on this!_

**Comment by tayloramurphy** [comment]
@aaronsteers Items found in my review:

- https://deploy-preview-663--meltano-hub.netlify.app/extractors/tap-auth0 doesn’t have the sidebar at any zoom level for me (tracked in https://github.com/meltano/hub/issues/785 I think?)
- Settings don’t detail the environment variable or any instructions on how to configure (compare https://hub.meltano.com/extractors/tap-auth0#client-id to https://deploy-preview-663--meltano-hub.netlify.app/extractors/tap-auth0#client_id-setting or compare https://deploy-preview-663--meltano-hub.netlify.app/utilities/great-expectations#ge_home-setting to https://hub.meltano.com/utilities/great_expectations#ge-home
- The installation and config instructions on most plugins don’t have any code blocks https://deploy-preview-663--meltano-hub.netlify.app/utilities/great-expectations they seem to be formatted for code, but it doesn’t have the background I would expect.
- Do we have the PyPI stats for these? I just noticed it missing on Great Expectations.
- On GE - I’m seeing a couple of wrong URLs. “pull request to the YAML file” goes to a non-existent section and same for the setting “No description provided”
- Under looking for help “If you’re having trouble getting the great_expectations extractor to work” - looks like we need to update to the plugin type (s/extractor/utility)
- On https://meltano-hub-gridsome.netlify.app/extractors/tap-google-analytics - it looks like we’re not rendering the images on https://hub.meltano.com/extractors/tap-google-analytics 
- Under “Maintainer” in the sidebar - I’d like to see the clean label and not the raw key. So “Meltano Community” instead of meltanolabs

commented in https://meltano.slack.com/archives/C043R4WV6TS/p1664225286228169?thread_ts=1664224307.638309&cid=C043R4WV6TS as well

**Comment by aaronsteers** [comment]
> @aaronsteers Items found in my review:
> 
> * [deploy-preview-663--meltano-hub.netlify.app/extractors/tap-auth0](https://deploy-preview-663--meltano-hub.netlify.app/extractors/tap-auth0) doesn’t have the sidebar at any zoom level for me (tracked in [fix(gridsome): plugin detail sidebar shouldn't float to bottom on small screen widths #785](https://github.com/meltano/hub/issues/785) I think?)

Should be resolved with #785

> * Settings don’t detail the environment variable or any instructions on how to configure (compare [hub.meltano.com/extractors/tap-auth0#client-id](https://hub.meltano.com/extractors/tap-auth0#client-id) to [deploy-preview-663--meltano-hub.netlify.app/extractors/tap-auth0#client_id-setting](https://deploy-preview-663--meltano-hub.netlify.app/extractors/tap-auth0#client_id-setting) or compare [deploy-preview-663--meltano-hub.netlify.app/utilities/great-expectations#ge_home-setting](https://deploy-preview-663--meltano-hub.netlify.app/utilities/great-expectations#ge_home-setting) to [hub.meltano.com/utilities/great_expectations#ge-home](https://hub.meltano.com/utilities/great_expectations#ge-home)

With `config --interactive` and just training people to configure things in general, I prefer not to have this everywhere on the page. Could be just once under "Settings > How to configure" with notes about the naming convention, `meltano config set` syntax and `--meltano config --interactive` option.

That said, if we see this as a blocker, we'll get it added back. It's tracked here:

- https://github.com/meltano/hub/issues/846

> * The installation and config instructions on most plugins don’t have any code blocks [deploy-preview-663--meltano-hub.netlify.app/utilities/great-expectations](https://deploy-preview-663--meltano-hub.netlify.app/utilities/great-expectations) they seem to be formatted for code, but it doesn’t have the background I would expect.

Confirmed.

https://deploy-preview-663--meltano-hub.netlify.app/utilities/great-expectations/#contribute

image

Contrasted with elsewhere on the page, where formatting matches expectations. (Possibly a distinction of how we use `pre` and `code`, or both.)

> * Do we have the PyPI stats for these? I just noticed it missing on Great Expectations.
> * On GE - I’m seeing a couple of wrong URLs. “pull request to the YAML file” goes to a non-existent section and same for the setting “No description provided”
> * Under looking for help “If you’re having trouble getting the great_expectations extractor to work” - looks like we need to update to the plugin type (s/extractor/utility)
> * On [meltano-hub-gridsome.netlify.app/extractors/tap-google-analytics](https://meltano-hub-gridsome.netlify.app/extractors/tap-google-analytics) - it looks like we’re not rendering the images on [hub.meltano.com/extractors/tap-google-analytics](https://hub.meltano.com/extractors/tap-google-analytics)
> * Under “Maintainer” in the sidebar - I’d like to see the clean label and not the raw key. So “Meltano Community” instead of meltanolabs
> 
> commented in [meltano.slack.com/archives/C043R4WV6TS/p1664225286228169?thread_ts=1664224307.638309&cid=C043R4WV6TS](https://meltano.slack.com/archives/C043R4WV6TS/p1664225286228169?thread_ts=1664224307.638309&cid=C043R4WV6TS) as well

**Comment by tayloramurphy** [comment]
> On GE - I’m seeing a couple of wrong URLs. “pull request to the YAML file” goes to a non-existent section and same for the setting “No description provided”

@aaronsteers I see they go to the #contributing header but that does feel a bit weird. My intuition was that the "pull request to the YAML file" would just go to the YAML file. And "No description provided" doesn't seem like it should be a link (but I guess the CTA is please edit it and add one!) I guess this is fine as-is and not really a blocker. 

>  Under looking for help “If you’re having trouble getting the great_expectations extractor to work” - looks like we need to update to the plugin type (s/extractor/utility)

I think this one got missed in your above comment from what I see. 

> With config --interactive and just training people to configure things in general, I prefer not to have this everywhere on the page. Could be just once under "Settings > How to configure" with notes about the naming convention, meltano config set syntax and --meltano config --interactive option.

I'll leave an additional comment in https://github.com/meltano/hub/issues/846 on my thoughts on the "How to Configure" section specifically 👍

**Comment by aaronsteers** [comment]
> > On GE - I’m seeing a couple of wrong URLs. “pull request to the YAML file” goes to a non-existent section and same for the setting “No description provided”
> 
> @aaronsteers I see they go to the #contributing header but that does feel a bit weird. My intuition was that the "pull request to the YAML file" would just go to the YAML file. And "No description provided" doesn't seem like it should be a link (but I guess the CTA is please edit it and add one!) I guess this is fine as-is and not really a blocker.
> 
> > Under looking for help “If you’re having trouble getting the great_expectations extractor to work” - looks like we need to update to the plugin type (s/extractor/utility)
> 
> I think this one got missed in your above comment from what I see.
> 
> > With config --interactive and just training people to configure things in general, I prefer not to have this everywhere on the page. Could be just once under "Settings > How to configure" with notes about the naming convention, meltano config set syntax and --meltano config --interactive option.
> 
> I'll leave an additional comment in #846 on my thoughts on the "How to Configure" section specifically 👍

I think all of these are addressed now in the pending PR:

- #851

**Review by tayloramurphy — APPROVED** [review]
Le

**Review by tayloramurphy — APPROVED** [review]
YAY! Ship it :D

**Pull request merged**
